goodmap/db.py

```
@@ -113,10 +555,91 @@ def json_db_get_locations(self, query, location_model):
     return get_locations_list_from_raw_data(self.data, query, location_model)
 
 
+def mongodb_db_get_locations(self, query, location_model):
```

Ruff (0.13.1)

Unused function argument: `location_model`

(ARG001)